src/item.cpp

```diff
@@ -8674,8 +8671,8 @@ bool item::made_of_any_food_components( bool deep_search ) const
     for( const std::pair<itype_id, std::vector<item>> pair : components ) {
         for( const item &it : pair.second ) {
             const auto &maybe_food = it.get_comestible();
-            bool must_be_food = maybe_food && ( maybe_food->default_nutrition.kcal() > 0 ||
-                                                !maybe_food->default_nutrition.vitamins().empty() );
+            bool must_be_food = maybe_food && ( maybe_food->default_nutrition_read_only().kcal() > 0 ||
```

Comment:
We actually shouldn't be using default_nutrition here, but I only realized that *after* I opened this PR.

Since it's out of scope I plan to make a followup PR to address this issue.